README.md

```diff
@@ -1,4 +1,6 @@
-# Strimzi: Kafka as a Service
+![Strimzi](./documentation/logo/strimzi.png)
+
+# Running Apache Kafka on Kubernetes and OpenShift
```

Comment:
I would say "Run" rather than "Running" because this isn't a formal procedure or anything.

I suggest we change "OpenShift" to "OpenShift Container Platform". I would like to make this change throughout the documentation when I get time.

Reply:
I will change into "Run" but OCP is the product name while we want to have the community one upstream (OpenShift Origin which is known as OKD today)

Reply:
Oh right, that makes sense! :)